Dockerfile.tor-onionbalance-manager

```diff
@@ -23,6 +25,7 @@ RUN apk add --no-cache --update \
         python3=3.10.11-r0 \
     && python3 -m pip install --no-cache-dir git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${VERSION}
```

Comment:
VERSION => OB_VERSION

Reply:
Just committed the fix, let's see the new build